lib/datadog/tracing/contrib/active_record/events/sql.rb

```diff
@@ -58,6 +58,9 @@ def process(span, event, _id, payload)
                 Contrib::Analytics.set_sample_rate(span, configuration[:analytics_sample_rate])
               end
 
+              # Measure service stats
+              Contrib::Analytics.set_measured(span)
```

Comment:
@zarirhamza, setting `measured` for spans that can be service-entry spans (can have a different service name than their parent span) causes double counting of trace metrics. Because of this, we were instructed to not set `measured` on these kind of spans.
Do you know if this has changed? If not, this proposed change will cause double counting of trace metrics for the spans in this PR.

Reply:
With the service naming changes, in v1, these spans would no longer be service entry spans since all spans would default to a global value. I was under the impression that setting a measured tag on a service entry span would not result in any problem, because the agent should have logic against double calculation of metrics on the same span? Do you have any reference as to why this wouldn't be the case?